packages/client/manifest.json

```diff
@@ -2346,6 +2346,43 @@
         "key": "disabled",
         "defaultValue": false
       },
+      {
+        "type": "select",
+        "label": "Type",
+        "key": "optionsType",
+        "defaultValue": "select",
+        "placeholder": "Pick an options type",
+        "options": [
+          {
+            "label": "Select",
+            "value": "select"
+          },
+          {
+            "label": "Checkboxes",
+            "value": "checkbox"
+          }
+        ]
+      },
+      {
+        "type": "select",
+        "label": "Direction",
+        "key": "direction",
+        "defaultValue": "vertical",
+        "options": [
+          {
+            "label": "Horizontal",
+            "value": "horizontal"
+          },
+          {
+            "label": "Vertical",
+            "value": "vertical"
+          }
+        ],
+        "dependsOn": {
+          "setting": "optionsType",
+          "value": "checkbox"
+        }
```

Comment:
NAB: Perfect usage of "dependsOn" :+1: One other setting which we need to adjust now is the "autocomplete" setting, which now is only relevant for the "select" variant.

Reply:
Good catch! I've added that to the manifest.